### DISCUSSION TITLE
keywords are not stored every time

### DISCUSSION THREAD

**Discussion by magnum61** [firefly-iii/firefly-iii#6351] [General]
I have installed Firefly fresh today and started logging the first transaction. 
I found the tag option pretty useful and started creating new keywords during the transaction. 

But i have identified that these tags are not stored as new in the keyword list. That happens not all the time, but random. 

After creating the tags using the dedicated function works, then the tags are stored and i can use them in new or existing transactions.

Any thoughts why it's not working every time in the transaction itself?

Thanks in advance

**Comment by JC5** [comment]
What do you mean with keywords?

**Reply by magnum61** [reply]
Sorry, i was using the interface in my language. I mean the "Tags"

**Reply by JC5** [reply]
Ah yes. Make sure you press enter after every entry. Otherwise they won't be saved.

**Comment by magnum61** [comment]
Tested it on a fresh install. It works as you described. 

Thanks
